pkgs/tools/admin/gimme-aws-creds/default.nix

```diff
@@ -0,0 +1,106 @@
+{ lib
+, python3
+, fetchFromGitHub
+, nix-update-script
+, testers
+, gimme-aws-creds
+}:
+
+let
+  py = python3.override {
+    packageOverrides = self: super: {
+      fido2 = super.fido2.overridePythonAttrs (oldAttrs: rec {
+        version = "0.9.3";
+        src = self.fetchPypi {
+          inherit (oldAttrs) pname;
+          inherit version;
+          hash = "sha256-tF6JphCc/Lfxu1E3dqotZAjpXEgi+DolORi5RAg0Zuw=";
+        };
+      });
+
+      okta = super.okta.overridePythonAttrs (oldAttrs: rec {
```

Comment:
Wait, why are initiating okta 2.8.0 but then pulling a much older version here?

Reply:
`gimme-aws-creds` uses a really outdated version of okta lib.
I wanted to create the derivation for okta at the current version and adapt it for `gimme-aws-creds`
I don't think a lot of project will use such an old okta version.

Reply:
If it's the wrong way to do it, I would gladly modify that.

Reply:
No I just wanted to make sure there was a good reason. I think your reasoning sounds fine.